src/modules/client/pages/ni/config/CompanyConfig.vue

```diff
@@ -284,7 +284,7 @@ export default {
   computed: {
     ...mapGetters({ clientRole: 'main/clientRole' }),
     canUpdateErpConfig () {
-      const ability = defineAbilitiesFor(this.clientRole, null, this.company);
+      const ability = defineAbilitiesFor({ clientRole: this.clientRole, company: this.company });
```

Comment:
idem ici tu peux envoyer this.loggedUser (avec un pick sur les champs pour ne pas encoyer la terre entiere)